docs/components/saas/byok/cost-and-troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+---
+id: cost-and-troubleshooting
+title: "Cost implications and troubleshooting"
+description: "Learn more about cost implications and troubleshooting."
+---
+
+Using external encryption keys with **Amazon KMS** incurs costs directly in your Amazon account. Camunda does not charge for the feature itself, but you are responsible for all Amazon KMS usage.
+
+## Cost implications
```

Comment:
I suggest moving this section under the main BYOK page, as it is an important part of the configuration.

Reply:
Cost and troubleshooting do not seem to be related to each other, ie my main point. I think moving cost to main article would be more fitting for this info.